Sources/SwiftLibraryPluginProvider/LibraryPluginProvider.swift

```diff
@@ -33,6 +35,8 @@ import SwiftSyntaxMacros
 @_implementationOnly import Glibc
 #elseif canImport(Musl)
 @_implementationOnly import Musl
+#elseif canImport(Android)
+@_implementationOnly import Android
 #endif
```

Comment:
I think that we should hoist the android check above glibc, because the older releases had glibc on android as well.